profiles/bieefilled.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/65086865/243136312-eeef6c18-3f9e-492c-a224-83e4ed4dccfc.png",
+  "issueId": 1835,
+  "name": "Peter Samuel",
+  "username": "Bieefilled"
```

I believe this should be `Petsamuel`.
```suggestion
  "username": "Petsamuel"
```

---

I own that name too